lib/validator.js

```diff
@@ -288,6 +288,11 @@ Specberus.prototype.getDocumentDate = function () {
     const element = this.jsDocument.querySelector(
         'body div.head h2:not(#subtitle)'
     );
+    if (!element) {
```

Comment:
Adding the error here makes it more difficult to test with the current test suite (note that the test suite is broken with that update).
It seems more appropriate to update `lib/rules/metadata/profile.js` and `lib/rules/metadata/docDate.js` instead which are the rules used when selecting the profile 'auto'.
https://github.com/w3c/specberus/blob/main/lib/l10n-en_GB.js#L26 should also be updated to provide the same kind of details.